environments/tipstaff.json

```diff
@@ -0,0 +1,30 @@
+{
+    "account-type": "member",
+    "environments": [
+      {
+        "name": "development",
+        "access": [
+          {
+            "github_slug": "tipstaff-team",
+            "level": "sandbox",
+            "nuke": "rebuild"
+          }
+        ]
+      },
+           {
```

Comment:
The indentation/formatting looks a bit funny here. I think it is correct in terms of the syntax, but it's difficult to review. 

I know we have a formatter that runs nightly now, but in the future you can format your JSON code in VSC with `Ctrl+Shift+I`. It's a good habit to have anyway.